File: NeoPlayerTestClient/MediaData.cs
namespace NeoPlayerTestClient
{
	public class MediaData
	{
		public string Description { get; private set; }
		public string URL { get; private set; }
		public bool IsQueued { get; private set; }

		public MediaData(string description, string url, bool isQueued)
		{
			Description = description;
			URL = url;
			IsQueued = isQueued;
		}
	}
}

File: NeoRemote/app/src/main/java/neoplayer/neoremote/MediaData.java
package neoplayer.neoremote;

public class MediaData {
    public String description;
    public String url;
    public long playlistOrder;

    public MediaData(String description, String url, long playlistOrder) {
        this.description = description;
        this.url = url;
        this.playlistOrder = playlistOrder;
    }
}